pkg/cli/init-templates/.dockerignore

```diff
@@ -0,0 +1,5 @@
+# https://docs.docker.com/engine/reference/builder/#dockerignore-file
+.DS_Store
+.git
+.github
+.gitignore
```

Comment:
Are there some other good defaults we could add here?

cc @replicate/models

Reply:
@zeke I just updated this based on [other `.dockerignore` files in our org](https://github.com/search?q=org%3Areplicate+path%3A.dockerignore&type=code).